napari/layers/points/points.py

```diff
@@ -991,30 +913,12 @@ def size(self) -> np.ndarray:
     def size(self, size: float | np.ndarray | list) -> None:
         try:
             self._size = np.broadcast_to(size, len(self.data)).copy()
-        except ValueError as e:
-            # deprecated anisotropic sizes; extra check should be removed in future version
-            try:
-                self._size = np.broadcast_to(
-                    size, self.data.shape[::-1]
-                ).T.copy()
-            except ValueError:
-                raise ValueError(
-                    trans._(
-                        'Size is not compatible for broadcasting',
-                        deferred=True,
-                    )
-                ) from e
-            else:
-                self._size = np.mean(size, axis=1)
-                warnings.warn(
-                    trans._(
-                        'Since 0.4.18 point sizes must be isotropic; the average from each dimension will be'
-                        ' used instead. This will become an error in version 0.6.0.',
-                        deferred=True,
-                    ),
-                    category=DeprecationWarning,
-                    stacklevel=2,
-                )
+except ValueError as e:
+    raise ValueError(
+        trans._(
+            'Size is not compatible for broadcasting (may be anisotropic)', deferred=True,
+         )
+     )
```

Comment:
```suggestion
        except ValueError as e:
            raise ValueError(
                trans._(
                    'Size is not compatible for broadcasting (may be anisotropic)', deferred=True,
                 )
             )
```
This should fix the indents.